integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonCreateDataMapCommand.scala

```diff
@@ -60,7 +61,14 @@ case class CarbonCreateDataMapCommand(
     } else {
       dmProperties
     }
-
+    val dataMapProvider = {
+      try {
+        DataMapProvider.getDataMapProvider(dmClassName)
```

Comment:
suggest change "dmClassName" to "providerName"

Reply:
it could be provider name or classname, user can give any one.

Reply:
I understand user can provide className and shortName, both belong to provider name, right?